### PR TITLE
[bugfix] Directly connect to node 0

### PR DESCRIPTION
**Pull request by minsulee2**
Because a load balancer does not work somehow, connect to the initial peer directly with ip address.

### CODE REVIEW

File: start_node_genesis_gcp.sh [hunk 1]
```diff
@@ -134,15 +134,19 @@ export GENESIS_CONFIGS_DIR=genesis-configs/testnet
...
     export TRACKER_WS_ADDR=ws://35.194.172.106:5000
+    export P2P_PEER_CANDIDATE_URL="http://35.194.169.78:8080/json-rpc"
     KEYSTORE_DIR=testnet_prod_keys
 elif [[ $SEASON = 'staging' ]]; then
     export TRACKER_WS_ADDR=ws://35.221.150.73:5000
+    export P2P_PEER_CANDIDATE_URL="http://35.194.139.219:8080/json-rpc"
 elif [[ $SEASON = 'dev' ]]; then
   if [[ $SHARD_INDEX -gt 0 ]]; then
     export GENESIS_CONFIGS_DIR=genesis-configs/sim-shard
+    export P2P_PEER_CANDIDATE_URL="http://35.194.235.180:8080/json-rpc"
```
Comment (platfowner) [inline start_node_genesis_gcp.sh:149]: this should be moved to near round line 153?

File: start_node_genesis_gcp.sh [hunk 2]
```diff
@@ -241,7 +245,8 @@ export BLOCKCHAIN_DATA_DIR="/home/ain_blockchain_data"
 # NOTE(liayoo): This is a temporary setting. Remove once domain is set up for afan metaverse related services.
 export CORS_WHITELIST=*
 printf "CORS_WHITELIST=$CORS_WHITELIST\n"
-export P2P_PEER_CANDIDATE_URL="https://$SEASON-api.ainetwork.ai/json-rpc"
+# FIXME(minsulee2, platfowner): Uncomment again when the load-balancer is updated.
+# export P2P_PEER_CANDIDATE_URL="https://$SEASON-api.ainetwork.ai/json-rpc"
```
Comment (platfowner) [inline start_node_genesis_gcp.sh:249]: I think api url cannot be used in bootstrapping. what about just removing these lines?

File: start_node_genesis_gcp.sh [hunk 3]
```diff
@@ -134,15 +134,19 @@ export GENESIS_CONFIGS_DIR=genesis-configs/testnet
 KEYSTORE_DIR=testnet_dev_staging_keys
 if [[ $SEASON = 'spring' ]]; then
     export TRACKER_WS_ADDR=ws://35.221.137.80:5000
+    export P2P_PEER_CANDIDATE_URL="http://35.221.184.48:8080/json-rpc"
```
Comment (platfowner) [inline start_node_genesis_gcp.sh:137]: could you also update start_node_incremental_gcp.sh?

File: start_node_genesis_gcp.sh (hunk 1 above)
Comment (minsulee2) [reply start_node_genesis_gcp.sh:149]: Oh! Moved!

File: start_node_genesis_gcp.sh (hunk 2 above)
Comment (minsulee2) [reply start_node_genesis_gcp.sh:249]: Got this!

File: start_node_genesis_gcp.sh (hunk 3 above)
Comment (minsulee2) [reply start_node_genesis_gcp.sh:137]: Done!

**Comment by minsulee2** [comment]
PTAL!

File: start_node_incremental_gcp.sh [hunk 4]
```diff
@@ -84,13 +84,17 @@ export GENESIS_CONFIGS_DIR=genesis-configs/testnet
...
     KEYSTORE_DIR=testnet_prod_keys
 elif [[ $SEASON = 'summer' ]]; then
     export TRACKER_WS_ADDR=ws://35.194.172.106:5000
+    export P2P_PEER_CANDIDATE_URL="http://35.194.169.78:8080/json-rpc"
     KEYSTORE_DIR=testnet_prod_keys
 elif [[ $SEASON = 'staging' ]]; then
     export TRACKER_WS_ADDR=ws://35.221.150.73:5000
+    export P2P_PEER_CANDIDATE_URL="http://35.194.139.219:8080/json-rpc"
 elif [[ $SEASON = 'dev' ]]; then
+    export P2P_PEER_CANDIDATE_URL="http://35.194.235.180:8080/json-rpc"
```
Comment (platfowner) [inline start_node_incremental_gcp.sh:97]: this line should be moved to around line 99?

File: start_node_genesis_gcp.sh [hunk 5]
```diff
@@ -134,16 +134,20 @@ export GENESIS_CONFIGS_DIR=genesis-configs/testnet
...
+    export P2P_PEER_CANDIDATE_URL="http://35.194.169.78:8080/json-rpc"
     KEYSTORE_DIR=testnet_prod_keys
 elif [[ $SEASON = 'staging' ]]; then
     export TRACKER_WS_ADDR=ws://35.221.150.73:5000
+    export P2P_PEER_CANDIDATE_URL="http://35.194.139.219:8080/json-rpc"
 elif [[ $SEASON = 'dev' ]]; then
   if [[ $SHARD_INDEX -gt 0 ]]; then
     export GENESIS_CONFIGS_DIR=genesis-configs/sim-shard
   fi
+    export P2P_PEER_CANDIDATE_URL="http://35.194.235.180:8080/json-rpc"
```
Comment (platfowner) [inline start_node_genesis_gcp.sh:150]: this line should be moved after line 153?

File: start_node_incremental_gcp.sh (hunk 4 above)
Comment (minsulee2) [reply start_node_incremental_gcp.sh]: Done

File: start_node_genesis_gcp.sh (hunk 5 above)
Comment (minsulee2) [reply start_node_genesis_gcp.sh]: Done!

**Comment by minsulee2** [comment]
PTAL. I've got a question why P2P_PEER_CANDIDATE_URL should be in `$SHARD_INDEX = 0`.

**Review by platfowner — APPROVED** [review]
LGTM
Thanks for fixing this!

**Comment by minsulee2** [comment]
Thanks for the quick review! Merging now for seeing what is going on!!

**Pull request merged**